pages/hackathon-grant.js

```diff
@@ -0,0 +1,221 @@
+import { Box, Container, Flex, Grid, Heading } from 'theme-ui'
+import Meta from '@hackclub/meta'
+import Head from 'next/head'
+import ForceTheme from '../components/force-theme'
+import Nav from '../components/nav'
+import Footer from '../components/footer'
+import Sparkles from '../components/sparkles'
+import NextLink from 'next/link'
+import { Link, Text, Button, Card } from 'theme-ui'
+import Icon from '@hackclub/icons'
+import Form from '../components/hackathon-grant/form'
+
+const Requirement = ({ title, children }) => {
+  return (
+    <Card
+      variant="primary"
+      sx={{
+        backgroundColor: 'darkless',
+        color: 'snow',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'start'
+      }}
+    >
+      <Flex sx={{ alignItems: 'center' }}>
+        <Box mr={3} sx={{ lineHeight: 0, flexShrink: 0 }}>
+          <Icon glyph="checkmark" />
+        </Box>
+        <Text variant="heading" sx={{ fontSize: 24, lineHeight: 1.5 }}>
+          {title}
+        </Text>
+      </Flex>
+
+      <Text pl={48} mt={2} sx={{ fontSize: 2 }}>
+        {children}
+      </Text>
+    </Card>
+  )
+}
+
+const HackathonGrant = () => {
+  return (
+    <>
+      <Box as="main" key="main">
+        <Nav dark />
+        <ForceTheme theme="dark" />
+        <Meta as={Head} title="Hackathon Grant" />
+        <Box
+          sx={{
+            pt: [5, null, null, null, 6],
+            pb: [3, 4, 5, null, 6],
+            textAlign: 'center',
+            backgroundImage:
+              "linear-gradient(to bottom, rgba(0, 0, 0, 0.8), rgba(0, 0, 0, 0.4)), url('https://cloud-3aeson6ue-hack-club-bot.vercel.app/0lahacks3.jpeg')",
+            backgroundSize: 'cover',
+            backgroundPosition: 'center center'
+          }}
+        >
+          <Box sx={{ width: '100%', maxWidth: '42rem', mx: 'auto' }}>
+            <Heading
+              sx={{
+                textAlign: 'center',
+                mt: [2, 4],
+                textShadow: '0 0 16px rgba(0, 0, 0, 1)'
+              }}
+              as="h1"
+              variant="title"
+            >
+              <Flex
+                sx={{ justifyContent: 'center', alignItems: 'center', mb: 2 }}
+              >
+                <Box
+                  width={64}
+                  height={64}
+                  sx={{
+                    width: 72,
+                    height: 72,
+                    backgroundImage:
+                      "url('https://assets.hackclub.com/icon-square.svg')",
+                    backgroundSize: 'contain',
+                    backgroundPosition: 'center center',
+                    backgroundRepeat: 'no-repeat',
+                    mr: 3,
+                    borderRadius: 8
+                  }}
+                ></Box>
+                <Box mr={3} opacity={0.75} sx={{ fontWeight: 'normal' }}>
+                  +
```

Comment:
the plus sign here has this weird coloring opacity thing on Safari

Reply:
<img width="242" alt="image" src="https://user-images.githubusercontent.com/28282096/184502765-0c91ef8e-06e2-4a32-ae6f-0d5a7bee0fa2.png">